### PR TITLE
Intermediate InaSAFE and Other Hazard Training Materials

### PR DESCRIPTION
**Pull request by adissadis**
hi @Charlotte-Morgan @rduivenvoorde @timlinux 

Please review my PR for intermediate InaSAFE and other hazard training material. I build local run and its already error-free. 

Thank you.

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
Hi @adissadis  @iyan31 - its great to see you got this file into rst but I really believe it should be reviewed by Tyler before it is merged because there is a lot of work required on the English.
As we discussed the other day, this is Tyler's role to assist HOT ID this. Then other people in the project can comment and help with minor edits as needed.

Hopefully Tyler is commenting on the google doc and you can update text strings as he modifies them.

**Comment by iyan31** [comment]
Hi @Charlotte-Morgan, as discussed on Friday, I already reviewed the English for the Intermediate before Adityo then change it to .rst. I will discuss with Tyler to focus on Other Hazard and re-check the Intermediate InaSAFE.

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+Introduction
+------------
+
+In `Run Basic InaSAFE <http://docs.inasafe.org/en/training/socialisation/run_basic_inasafe.html/>`__ we learned how to run flood analysis on
+population and buildings, understand flood impact default settings and
+impact summary report, and how to change analysis threshold. We also
+learned how to save our work and generate a pdf of the analysis results.
+In this exercise we will learn more techniques such as how to run
+|project_name| with aggregation data to produce more specific results. And we
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: to produce reports broken down by districts or regions.

Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: ``And we`` -->  ``We`` (Don't start a sentence with 'and')

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+will learn how to download data using the OpenStreetMap downloader and
+how to run |project_name| analysis on the downloaded building points and roads.
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: building points or building polygons?

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+------------------
+
+To improve participant’s understanding of the |project_name| function and
+features. By the end of this exercise, participants will be able to:
+
+- Define keywords using the |project_name| keyword wizard
+
+- Run |project_name| with aggregation data
+
+- Use the OpenStreetMap Downloader to download data for |project_name| and
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: remove terminating and

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+--------
+
+Run |project_name| for Population Data
+......................................
+
+A. Add and symbolize vector data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before we start, click :guilabel:`Open` and select :file:`DKI Jakarta Intermediate.qgs`.
+If there is notification to save the current project, you can choose
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: If there is **a** notification

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+
+.. image:: /static/training/socialisation/intermediate_inasafe_01.*
+   :align: center
+   :width: 300 pt
+
+Next, add :file:`Jakarta Flood 18113 WGS84` data from :file:`Run Intermediate InaSAFE > DKI Jakarta > 01 Hazard Data`. This is the flood hazard data
+for Jakarta from an actual flood event. During Jakarta floods in January
+2013, provincial disaster manager collects information about flood,
+including the location of flooded areas by sub-village boundary
+(`see more about this data at… <http://docs.inasafe.org/en/training/socialisation/datasets.html>`__).
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: We should use internal document references here. See http://sphinx-doc.org/markup/inline.html

File: docs/source/training/socialisation/run_intermediate_inasafe.rst [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+
+Let us symbolize this layer so that it shows only affected areas (if you
+forget how to symbolize, you can go to `Introduction to QGIS <http://docs.inasafe.org/en/training/socialisation/introduction_to_qgis.html/>`__).
```
Comment (timlinux) [inline docs/source/training/socialisation/run_intermediate_inasafe.rst]: as above - use :ref: and `section reference` for internal references http://sphinx-doc.org/markup/inline.html

**Comment by timlinux** [comment]
Hi

@iyan31 I think it still needs a general english review. I added some other notes above too and will review in more detail following the english clean up...

Regards

Tim

**Comment by iyan31** [comment]
@timlinux I will leave it to native English speaker then :). Tyler just told me this morning that he is just finished with reviewing Intermediate InaSAFE and will continue with Other Hazard, @adissadis will integrate the changes soon.

**Comment by TylerRadford** [comment]
Completed English review of Basic InaSAFE, Intermediate InaSAFE, and Other Hazards

**Comment by timlinux** [comment]
Thanks @TylerRadford .. I will take a look through for content issues now...

File: docs/source/training/socialisation/faq.rst [hunk 7]
```diff
@@ -74,8 +74,8 @@ QGIS and |project_name|.
 
     - Yes you can, but you need to enable on the fly projection.
     - In QGIS, activate OTF CRS Transformation
-       - Project ? Project Properties ? CRS ?
-       - checkbox in the Enable �on the fly� CRS transformation.
+       - Project > Project Properties > CRS
```
Comment (timlinux) [inline docs/source/training/socialisation/faq.rst]: :menuselection: `Project --> Project Properties --> CRS`

See http://sphinx-doc.org/markup/inline.html#role-menuselection

File: docs/source/training/socialisation/faq.rst [hunk 8]
```diff
@@ -74,8 +74,8 @@ QGIS and |project_name|.
...
+       - checkbox in the Enable 'on the fly' CRS transformation.
```
Comment (timlinux) [inline docs/source/training/socialisation/faq.rst]: Tick the :guilabel:`Enable on the fly transformation` checkbox 

See http://sphinx-doc.org/markup/inline.html#role-guilabel

**Comment by TylerRadford** [comment]
@timlinux probably should have mentioned - my edits were all in the Google Docs versions

**Comment by iyan31** [comment]
Hi @TylerRadford, @adissadis converting your edits in Google docs to .rst. We already integrate your inputs on Intermediate InaSAFE and planned to do the Other Hazard yesterday unfortunately the internet was down the the whole afternoon. We will work on it today

**Comment by adissadis** [comment]
@timlinux Hi tim, 

I already change the rst file based on Tyler inputs.

Already ran local build and only found one error that i don't know how to resolve it (already tried several way but still appear a warning)

image

**Comment by timlinux** [comment]
Hi

To fix the error change  ``training/socialisation/introduction_to_qgis.rst`` from

```
.. _introduction_to_qgis:


.. image:: /static/training/socialisation/inasafe_logo.*

Introduction to QGIS
====================
```

to

```
.. image:: /static/training/socialisation/inasafe_logo.*

.. _introduction_to_qgis:

Introduction to QGIS
====================
```

The section label must come immediately before the section header. Hope that makes sense?

**Comment by adissadis** [comment]
@timlinux @rduivenvoorde okay thank tim. I already fixed that. Now it's warning free and waiting for merge

**Comment by timlinux** [comment]
@adissadis does this PR already incorporate the english language fixes from @TylerRadford from your google doc? I think we must make one more pass through it to clean up the grammar. Why don't you go ahead and merge this PR and I will make a follow up one that does that?

**Comment by adissadis** [comment]
@timlinux yes its already incorporate english language fixed from @TylerRadford. Ok i will merge this PR. Thank you

**Pull request merged**